.env

```diff
@@ -31,13 +31,13 @@ SOLR_ADMIN_PASSWORD=SolrRocks
 SOLR_ADMIN_USER=solr
 SOLR_CLOUD_BOOTSTRAP=yes
 SOLR_COLLECTION=hydra-development
-SOLR_COLLECTION_NAME="${SOLR_COLLECTION}"
+SOLR_COLLECTION_NAME=hydra-development
```

Comment:
I "downed" all my containers and ran `main` with only your docker-compose.yml changes. it worked!

I used `sc` instead of `docker compose`, but they are the same thing under the hood so I don't think that's a factor.

Reply:
```suggestion
SOLR_COLLECTION_NAME="${SOLR_COLLECTION}"
```